templates/zerver/api/changelog.md

```diff
@@ -18,6 +18,10 @@ clients should check the `zulip_feature_level` field, present in the
 /register`](/api/register-queue) responses, to determine the API
 format used by the Zulip server that they are interacting with.
 
+**Feature level 125**
+
+* [`GET /bot-storage`](/api/get-storage): Added Documentation for `/GET bot storage`.
```

Comment:
Looking at the other changes in the  same file, there's a convention of using `"-"` instead of `"_"` as seen in `/api/update-settings`, `/api/get-subscriptions` etc.

> Remember that the url formats use underscores "_", not dashes or spaces.

Thanks for the review on the grammatical structure. I'll make the appropriate changes